vespa-athenz/src/main/java/com/yahoo/vespa/athenz/identityprovider/client/ZtsClient.java

```diff
@@ -82,6 +81,43 @@ public InstanceIdentity sendInstanceRefreshRequest(String providerService,
         }
     }
 
+    ZToken getRoleToken(AthenzDomain domain,
+                        URI ztsEndpoint,
+                        SSLContext sslContext) {
+         // TODO ztsEndpoint should contain '/zts/v1' as path
+        URI correctedZtsEndpoint = ztsEndpoint.resolve("/zts/v1");
+        return new ZToken(
+                new ZTSClient(correctedZtsEndpoint.toString(), sslContext)
+                        .getRoleToken(domain.getName()).getToken());
```

Comment:
Verify return from ZTSClient.getRoleToken when not member of role _null_ or _exception_?

Reply:
will return exception -> OK